### PR TITLE
Add read only views of the deployment modal steps

### PR DESCRIPTION
**Pull request by stojg**
This also includes some refactorings and adding sass variables for colours and
uses an individual sass file for the react component

### CODE REVIEW

**Comment by halkyon** [comment]
How does this differ from `BuildStatus.jsx` which is already in there?

**Comment by stojg** [comment]
Yup, plan is to merge them together somehow

File: js/containers/TargetRelease.jsx [hunk 1]
```diff
@@ -20,14 +22,18 @@ var targetRelease = function(props) {
...
+			<StatusBox type="default">
+				<a
+					href={props.deployment.commit_url}
+					title={props.deployment.sha}
+				>
+					{props.deployment.short_sha}
+				</a>
+				&nbsp;<span className="label label-default">{props.deployment.branch}</span>
+				&nbsp;{props.deployment.commit_message}<br/>
+				<small>XXX days old</small>
```
Comment (stojg) [inline js/containers/TargetRelease.jsx]: I ended up removing it until I had a chat with the designer.

**Comment by stojg** [comment]
Okay, I've done what I can to align the styling and output

File: js/components/StatusBox.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var React = require("react");
+
+function StatusBox (props) {
+	return (
+		<div className={"status-box "+props.type}>
+			{props.children}
+		</div>
+	)
```
Comment (halkyon) [inline js/components/StatusBox.jsx]: missing `;`

File: js/containers/CurrentBuildStatus.jsx [hunk 3]
```diff
@@ -8,29 +8,21 @@ const actions = require('../_actions.js');
...
-					</a>
-				</li>
-				<li>
-					<a className="info-action" onClick={() => props.onItemClick(props.deployment.id)}>
-						<i className="fa fa-info-circle"></i>
-					</a>
-				</li>
-			</ul>
+		<div className="current-build-status" >
+			<BuildStatus deployment={props.deployment} open={props.onItemClick} />
```
Comment (halkyon) [inline js/containers/CurrentBuildStatus.jsx]: `props.onItemClick` is for open prop?

**Pull request merged**